dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/test/java/org/apache/dolphinscheduler/plugin/task/api/utils/ParameterUtilsTest.java

```diff
@@ -113,4 +113,15 @@ public void testHandleEscapes() throws Exception {
         Assertions.assertEquals("test Parameter", ParameterUtils.handleEscapes("test Parameter"));
         Assertions.assertEquals("////%test////%Parameter", ParameterUtils.handleEscapes("%test%Parameter"));
     }
+
+    @Test
+    public void testProcessSqlQuestionMark() {
+        String querySql = "select id, concat('?', name) from test";
+        String expected = "select id, concat('@@QUESTION_UNIQUE_MARK@@', name) from test";
```

Comment:
maybe this unique mark could be a constant

Reply:
ok, i will replace it.